Parsers/ASimAuditEvent/Parsers/vimAuditEventVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,221 @@
+          | parse description_s with "Parameters updated for " temp_config2: string " (ID: " temp_configid2: string ") for policy with ID " temp_policyid2: string
+          | extend
+              temp_operation = coalesce(temp_config1, temp_config2),
+              temp_configid = coalesce(temp_configid1, temp_configid2)
+          | extend
+              EventType = "Set", 
+              Operation = strcat("Parameters updated for ", temp_operation, " for policy ", temp_policyname1, tostring(split(temp_policyid2, "{")[0])),
+              Object = strcat("Policy ", coalesce(temp_policyname1, temp_policyid2)),
+              ObjectType = "Policy Rule",
+              AdditionalFields = bag_pack("config id", temp_configid);
+      let Reputation = allData
+          | where description_s has_cs "Reputation"
+          | parse description_s with "User " * " " temp_type1: string " Reputation" * " for Organization ID " temp_orgid1: string " of type " temp_reptype1: string " to " temp_list1: string " with content: " temp_content1: string " | " temp_restmessage1: string
+          | parse description_s with "User " * " " temp_type2: string " Reputation" * " for Organization ID " temp_orgid2: string ": " temp_content2: string " | " temp_restmessage2: string
+          | extend
+              temp_type = coalesce(temp_type1, temp_type2),
+              Object = iff(isnotempty(temp_reptype1), strcat("Reputation Override of type ", temp_reptype1), "Reputation Override"),
+              ObjectType = "Configuration Atom"
+          | lookup EventTypeLookup on temp_type
+          | extend Operation = strcat(temp_type, " ", Object)
+          | extend AdditionalFields = bag_pack("reputation value", coalesce(temp_content1, temp_content2));
+      union
+          Enabled,
+          Set,
+          AlertNotify,
+          CustomRole,
+          Policy,
+          Changed,
+          ParamsUpdated,
+          Reputation
+      | extend EventResult = "Success"
+      | where (eventresult == "*" or EventResult =~ eventresult)
+          and (array_length(eventtype_in) == 0 or EventType has_any (eventtype_in))
+          and (array_length(operation_has_any) == 0 or Operation has_any (operation_has_any))
+          and (array_length(object_has_any) == 0 or Object has_any (object_has_any))
+      | extend
+          EventStartTime = unixtime_milliseconds_todatetime(eventTime_d),
+          EventSeverity = coalesce(EventSeverity1, "Informational"),
+          AdditionalFields = bag_merge(AdditionalFields, bag_pack("flagged", flagged_b))
+      | extend
+          EventProduct = "Carbon Black Cloud",
+          EventSchema = "AuditEvent",
+          EventSchemaVersion = "0.1",
+          EventVendor = "VMware",
+          DvcAction = "Allow",
+          EventCount = int(1)
+      | project-rename
+          ActorUsername = loginName_s,
+          EventUid = _ItemId,
+          SrcIpAddr = clientIp_s,
```

Comment:
can you validate this?
Src - The system used by the Actor to initiate the configuration operation, if different than Target.
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-audit

Reply:
In case of VMware Carbon Black Cloud, ip coming in data is of network from where user has performed audit activity.